graylog2-web-interface/src/components/common/URLWhitelistFormModal.jsx

```diff
@@ -59,7 +59,22 @@ class URLWhiteListFormModal extends React.Component<Props, State> {
     }
 
 
-  _getConfig = (configType: string) => {
+    componentDidUpdate() {
+      const { config: { entries } } = this.state;
+      const urlwhitelistConfig = this._getConfig(URL_WHITELIST_CONFIG);
+      if (urlwhitelistConfig && entries.length === 0) {
+        this._setDefaultWhiteListState(urlwhitelistConfig);
+      }
+    }
+
+  _setDefaultWhiteListState =(urlwhitelistConfig) => {
```

Comment:
I still think this function (or a strip down version of it) should be called from the constructor, so the initial state comes from the given props and it's not empty.

Reply:
As we discussed, the data is fetched in `componentDidMount()` and setting a default state until then is fine.